openedx_caliper_tracking/processor.py

```diff
@@ -107,12 +114,20 @@ def __call__(self, event):
 
             CALIPER_LOGGER.info(json.dumps(transformed_event))
 
-            if getattr(settings, 'CALIPER_DELIVERY_ENDPOINT') and getattr(settings, 'CALIPER_DELIVERY_AUTH_TOKEN'):
+            if (getattr(settings, 'CALIPER_DELIVERY_ENDPOINT')
+                    and getattr(settings, 'CALIPER_DELIVERY_AUTH_TOKEN')):
                 deliver_caliper_event(transformed_event, event.get('event_type'))
 
+            if (getattr(settings, 'ENALBE_KAFKA_FOR_CALIPER')
```

Comment:
```suggestion
            if (hasattr(settings, 'ENALBE_KAFKA_FOR_CALIPER')
```

Reply:
or use getattr(settings, 'ENALBE_KAFKA_FOR_CALIPER', False))

Reply:
done in the latest commit.